packages/client/src/v2-events/features/events/ReviewSection.tsx

```diff
@@ -0,0 +1,552 @@
+  padding: 0px 32px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    padding: 0;
+  }
+`
+const DeclarationDataContainer = styled.div``
+
+enum REJECT_ACTIONS {
+  ARCHIVE,
+  SEND_FOR_UPDATE
+}
+
+interface RejectionState {
+  rejectAction: REJECT_ACTIONS
+  details: string
+  isDuplicate: boolean
+}
+
+const ReviewSectionComponent = ({ event }: { event: EventDocument }) => {
+  const [modal, openModal] = useModal()
+
+  const { data } = event.actions.filter(
+    (action) => action.type === 'DECLARE'
+  )[0]
+
+  const { eventConfiguration: configuration } = useEventConfiguration(
+    event.type
+  )
+
+  if (!configuration) {
+    throw new Error('Event configuration not found with type: ' + event.type)
+  }
+
+  const { forms } = configuration.actions.filter(
+    (action) => action.type === 'DECLARE'
+  )[0]
+
+  const handleRegister = async () => {
+    const confirmedRegister = await openModal<boolean | null>((close) => (
+      <RegisterModal close={close}></RegisterModal>
+    ))
+    if (confirmedRegister) {
+      alert('Registered new member')
+    }
+    return
+  }
+
+  const handleReject = async () => {
+    const confirmedReject = await openModal<RejectionState | null>((close) => (
+      <RejectModal close={close}></RejectModal>
+    ))
+    if (confirmedReject) {
+      const { rejectAction, ...rest } = confirmedReject
+      switch (rejectAction) {
+        case REJECT_ACTIONS.ARCHIVE:
+          alert('Archived the registration ' + JSON.stringify(rest))
+          break
+        case REJECT_ACTIONS.SEND_FOR_UPDATE:
+          alert('Sent the registration for update ' + JSON.stringify(rest))
+          break
+        default:
+          break
+      }
+    }
+    return
+  }
+
+  const handleEdit = async (
+    e: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement, MouseEvent>,
+    fieldId: string
+  ) => {
+    e.stopPropagation()
+    const confirmedEdit = await openModal<boolean | null>((close) => (
+      <EditFieldModal close={close}></EditFieldModal>
+    ))
+    if (confirmedEdit) {
+      alert('Editing field: ' + fieldId)
+    }
+    return
+  }
+
+  return (
+    <Frame
+      skipToContentText="Skip to form"
+      header={<FormHeader label={configuration.label} />}
+    >
+      <Row>
+        <LeftColumn>
+          <Card>
+            <HeaderContainer>
+              <HeaderContent>
+                <Stack
+                  direction="column"
+                  alignItems="flex-start"
+                  justify-content="flex-start"
+                  gap={6}
+                >
+                  <TitleContainer id={`header_title`}>
+                    {configuration.label.defaultMessage}
+                  </TitleContainer>
+                  <SubjectContainer id={`header_subject`}>
+                    {'Member registration for ' +
+                      (data['applicant.firstname'] || '') +
+                      ' ' +
+                      (data['applicant.surname'] || '')}
```

Comment:
Would this title come from event configuration + handlebars for data?

Reply:
No. Event configuration does not have this title.